docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
   grafana:
     image: grafana/grafana-oss:${GRAFANA_VERSION:-latest}
     ports:
-      - "${GRAFANA_PORT:-3000}:3000"
+      - "${GRAFANA_BIND:-127.0.0.1}:${GRAFANA_PORT:-3000}:3000"
```

Comment:
Nice!  TIL that you can specify the HOST in these port bindings... https://github.com/compose-spec/compose-spec/blob/master/spec.md#ports

I do wonder about the default here, though, and am just trying to decide what would be least surprising for someone trying to deploy this.  Personally, I am leaning towards having `0.0.0.0` as the default since I would consider running behind a reverse proxy to be _an exceptional case_ and not the _expected default case._  By default, I think I would expect Grafana to be available directly on the port that I specify.  If I plan to use a reverse proxy to host Grafana, then I would expect to have to include some special config.

Open to having my mind changed on this though!

Reply:
@jkuester - thanks for the call just now!  I think we agreed that in development mode, running it locally, this will just work as expected and be avail at `http://locahost:3000`.  But when an admin goes to deploy it, you're right, it will NOT be available by default and they'll either need to set up the TLS terminating reverse proxy to `localhost:3000` or knowingly make it insecure by binding `3000` running `http` to the public IP.

Approve if you agree \o/

Reply:
Sorry for the delay!  Got pulled into another call.  Yes, this is good to go!